base/shell/explorer/lang/it-IT.rc

```diff
@@ -58,7 +58,7 @@ BEGIN
         MENUITEM "&Fine sessione %s...", IDM_LOGOFF, MFT_STRING, MFS_ENABLED
         MENUITEM "Disconnetti...", IDM_DISCONNECT, MFT_STRING, MFS_ENABLED
         MENUITEM "&Sblocca il computer", IDM_UNDOCKCOMPUTER, MFT_STRING, MFS_ENABLED
```

Comment:
Not "sistema" here?

Reply:
Is the same. If the translation need to be Windows friendly computer is better but that it means the same things.